action.yml

```
@@ -14,6 +14,10 @@ inputs:
     description: "Duration after which artifact will expire in days."
     required: false
     default: "1"
+outputs:
+  artifact-id:
```

Can we please make this snake-cased? All of the Pages first-party actions use snake-case for our input and outputs (with one unfortunate exception):

```suggestion
  artifact_id:
```